### PR TITLE
Update content for skin prep guide, patchy, shield, & BioAmp EXG Pill

### PR DESCRIPTION
**Pull request by bhawnasehgal**
@lorforlinux Please review & merge.

### CODE REVIEW

File: guides/usage-guides/skin-preparation/index.rst [hunk 1]
```diff
@@ -8,26 +8,35 @@ Why skin preparation is important?
...
+| `Nuprep gel`_                       | Mildly abrasive, highly conductive gel that should be applied before placing the electrodes on the skin to improve signal quality & enhances the performance of monitoring electrodes.                |
 +-------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| **NuPrep Gel**                      | Mildly abrasive, highly conductive gel that should be applied before placing the electrodes on the skin to improve signal quality & enhances the performance of monitoring electrodes.                |
+| `Electrode Gel`_                    | Highly conductive gel that acts as a coupling agent between electrodes and the skin to aid the transmission of biopotential signals like ECG, EMG, EOG, or EEG.                                       |
```
Comment (lorforlinux) [inline guides/usage-guides/skin-preparation/index.rst:28]:
```suggestion
| `Electrode Gel`_                    | Highly conductive gel that acts as a coupling agent between dry electrodes and the skin to aid the transmission of biopotential signals like ECG, EMG, EOG, or EEG.                                   |
```

File: guides/usage-guides/skin-preparation/index.rst [hunk 2]
```diff
@@ -36,52 +45,85 @@ You can follow the steps given below to do the skin preparation properly:
...
-.. image:: media/skin-prep-emg.*
+.. figure:: media/skin-prep-emg.*
+    :align: center
 
-.. image:: media/skin-prep-ecg.*
+    Target area to record EMG
 
-.. image:: media/skin-prep-eeg.*
+.. figure:: media/skin-prep-ecg.*
+    :align: center
```
Comment (lorforlinux) [inline guides/usage-guides/skin-preparation/index.rst:61]:
```suggestion
    :align: center
    
    Target area to record ECG
```

File: guides/usage-guides/skin-preparation/index.rst [hunk 3]
```diff
@@ -36,52 +45,85 @@ You can follow the steps given below to do the skin preparation properly:
...
-Now you can either use gel electrodes or the BioAmp bands for the signal recording.
+Now you can either use gel electrodes or BioAmp bands for the signal recording.
 
 Using gel electrodes
 ----------------------
 
-  Connect the BioAmp cable to gel electrodes, peel the plastic backing from electrodes and place the IN+, IN-, REF cables according to your specific biopotential recording.
+Connect the BioAmp cable to gel electrodes, peel the plastic backing from electrodes and place the IN+, IN-, REF cables according to your specific biopotential recording.
+
+.. figure:: media/step-5a.*
```
Comment (lorforlinux) [inline guides/usage-guides/skin-preparation/index.rst:105]: Should we add some figure alt text?

File: guides/usage-guides/skin-preparation/index.rst [hunk 4]
```diff
@@ -36,52 +45,85 @@ You can follow the steps given below to do the skin preparation properly:
...
+    :align: center
+
+.. note:: While placing the gel electrodes on the skin, make sure to place the non-sticky tab of the electrode in the direction opposite to your hair growth. This allows you to remove the electrodes easily without pulling off much body hair.
```
Comment (lorforlinux) [inline guides/usage-guides/skin-preparation/index.rst:108]: We need graphical representation here.

**Comment by lorforlinux** [comment]
@bhawnasehgal Minor changes suggested, please check.

File: guides/usage-guides/skin-preparation/index.rst (hunk 3 above)
Comment (bhawnasehgal) [reply guides/usage-guides/skin-preparation/index.rst:105]: Sure, adding it.

File: guides/usage-guides/skin-preparation/index.rst (hunk 4 above)
Comment (bhawnasehgal) [reply guides/usage-guides/skin-preparation/index.rst:108]: Yes, will be creating a separate guide on how to use gel electrodes under guides section and link it here.

Comment (bhawnasehgal) [reply guides/usage-guides/skin-preparation/index.rst:116]: Created an issue #44 for this

**Comment by bhawnasehgal** [comment]
I have mage the suggested changes. Please merge.

**Comment by lorforlinux** [comment]
@bhawnasehgal thank you for the contribution, I am merging this now.

File: guides/usage-guides/skin-preparation/index.rst (hunk 4 above)
Comment (bhawnasehgal) [reply guides/usage-guides/skin-preparation/index.rst:116]: Resolved in #46

**Pull request merged**